sdk/core/core-client-rest/review/core-client.api.md

```diff
@@ -98,7 +99,7 @@ export type HttpBrowserStreamResponse = HttpResponse & {
 
 // @public
 export type HttpNodeStreamResponse = HttpResponse & {
-    body?: NodeJS.ReadableStream;
+    body?: NodeJSReadableStream;
```

Comment:
Could we use `stream.Readable`? If possible it might be better than introducing our own interface

Reply:
Doing this opens us up to higher risk for breaking changes if Readable changes. Also, Readable is a class, if it gets private members, and you try to pass an object to a context that expects a Readable class type, you'll get a type error.